lib/glific/groups.ex

```diff
@@ -116,7 +116,7 @@ defmodule Glific.Groups do
   """
   @spec get_or_create_group_by_label(String.t(), non_neg_integer) :: {:ok, Group.t()} | nil
   def get_or_create_group_by_label(label, organization_id) do
-    case Repo.get_by(Group, %{label: label}) do
+    case Repo.get_by(Group, %{label: label}, skip_organization_id: true) do
```

Comment:
This is a generic function, which we are using all over the place? I don't think skip_organization_id should be here?
Can you create a new function wherever you need this? 

Also, in case two organizations will have the same collection it will always fetch the first one.

Reply:
@pankaj-ag added organization in Repo.get